src/python/hangman/README.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Lærerveiledning - Hangman
+level: 3
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal vi lage det klassiske spillet Hangman. Der brukeren prøver å gjette seg frem til et hemmelig ord. 
```

Comment:
komma istedenfor punktum, og slå sammen til en setning. pr nå er setning nr to ufullstendig